.github/workflows/reviewdog.yml

```diff
@@ -1,6 +1,7 @@
 name: reviewdog
 on: 
-  [pull_request]
+  pull_request:
+      types: [review_requested, ready_for_review]
```

Comment:
Could also take review_requested off. What do you think @theyokohamalife

Reply:
If we add back code owners maybe it's good to take off `review_requested`

Reply:
I think it would be a good idea to add code owners back now that the repo is public.

Reply:
It's so simple I am just making a PR and no issue: https://github.com/ourjapanlife/findadoc-server/pull/68